### PR TITLE
Safe code documentation

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
Adds better documentation and variable naming to safe code. Also removed one variable that hasn't done anything in years. 

## Why It's Good For The Game
Good documentation and good variable names makes better code

## Changelog
No player-facing changes.

### CODE REVIEW

File: code/game/objects/items/storage/secure.dm [hunk 1]
```diff
@@ -12,15 +12,23 @@
...
+	var/entered_code = ""
+	/// The code that will open this safe
+	var/lock_code = null
+	/// Does this lock have a code set?
+	var/lock_set = FALSE
+	/// Is this lock currently being hacked?
+	var/lock_hacking = FALSE
+	/// Is the safe service panel open?
+	var/panel_open = FALSE
+	/// Is this door hackable?
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/storage/secure.dm:31]: Please move new variables below inherited variables

File: code/game/objects/items/storage/secure.dm [hunk 2]
```diff
@@ -12,15 +12,23 @@
...
-	var/code = ""
-	var/l_code = null
-	var/l_set = FALSE
-	var/l_setshort = FALSE
-	var/l_hacking = FALSE
-	var/open = FALSE
+	/// The code entered by the user
+	var/entered_code = ""
+	/// The code that will open this safe
+	var/lock_code = null
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/storage/secure.dm:24]: Can you remove = null

File: code/game/objects/items/storage/secure.dm [hunk 3]
```diff
@@ -12,15 +12,23 @@
...
+	/// icon_state of opened safe
 	var/icon_opened = "secure0"
-	var/code = ""
-	var/l_code = null
-	var/l_set = FALSE
-	var/l_setshort = FALSE
-	var/l_hacking = FALSE
-	var/open = FALSE
+	/// The code entered by the user
+	var/entered_code = ""
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/storage/secure.dm:22]: Why not null? Cursed

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
🐱

File: code/game/objects/items/storage/secure.dm [hunk 4]
```diff
@@ -87,34 +95,33 @@
...
 		for(var/mob/M in viewers(1, loc))
 			if ((M.client && M.machine == src))
 				attack_self(M)
 			return
 	return
 
-
 // -----------------------------
 //        Secure Briefcase
 // -----------------------------
```
Comment (tralezab) [inline code/game/objects/items/storage/secure.dm:127]: get rid of the stupid fancy comment blocks

**Review by tralezab — CHANGES_REQUESTED** [review]

File: code/game/objects/items/storage/secure.dm (hunk 4 above)
Comment (tralezab) [reply code/game/objects/items/storage/secure.dm:127]: if there are any more in this file, blam them too.

**Pull request merged**